distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/CatalogBundle.java

```diff
@@ -195,8 +197,13 @@ public CatalogFramework getCatalogFramework() throws InterruptedException {
     }
 
     public void setFanout(boolean fanoutEnabled) throws IOException {
-        Map<String, Object> properties = adminConfig.getDdfConfigAdmin()
-                .getProperties(CATALOG_FRAMEWORK_PID);
+        Map<String, Object> properties = null;
```

Comment:
✏️ Should use the same idiom as in `setConfigProperty` with the `Optional` to avoid the null check after the try/catch.